src/site/content/en/payments/payments.11tydata.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  // Tags are inherited by all posts.
+  tags: ['pathItem', 'payments'],
+  path: {
+    // Slug is used by landing pages like / and /learn to link to this path.
+    // Because it affects urls, the slug should never be translated.
+    slug: 'payments',
+    cover: '/images/collections/payments.svg',
```

Comment:
image isn't loading because it's named `payment.svg` but this is using `payments.svg` (plural).

Reply:
Thanks for the heads up, must have crept up in a merge conflict. This is not ready for review at all, it was just an automatic review request. I'll ping you when I'm done :)